client/src/components/Home/banner.js

```diff
@@ -1,5 +1,5 @@
 import React from "react";
-import Title from "../UsernameForm/Title";
+// import Title from "../UsernameForm/Title";
```

Comment:
I believe the <Title> tag is added there for styling of Titles/Headers like "Recently Recommended" or "Saved Shows".  I would like to keep this.